Modules/Core/Common/include/itkLoggerOutput.h

```diff
@@ -91,6 +91,12 @@ class ITKCommon_EXPORT LoggerOutput : public OutputWindow
   void
   DisplayDebugText(const char * t) override;
 
+  // Bring base class overloads into scope to avoid hiding
```

Comment:
In commit message "there own" -> "their own".

Reply:
done